pr-checklist.md

```diff
@@ -0,0 +1,44 @@
+# Documentation checklist
+
+Before requesting a PR review, please check your work against this list of commonly missed items from the style guide. 
+
+Guidance for Grammarly, Screenshots, and Diagrams can be found in the [Docs Standards](https://papermoon.atlassian.net/wiki/spaces/DR/pages/885261/Docs+Standards){target=\_blank}
+
+- [ ] Check your lists
+	- [ ] Ensure lists requiring a specific order (step-by-step instructions, etc.) are numbered
+	- [ ] Use bullets for lists that can be in any order (list of features, etc.)
+	- [ ] Check your formatting for list items - they should look like:
+		- Item name - copy about the item/description
+			- [ ] Item name capitalized
+			- [ ] space, dash, space between item name and description
+			- [ ] the description starts with a lowercase letter
+			- [ ] there is no punctuation at the end of the description
+- [ ] Check your links
+	- [ ] Test all links to ensure they open to the expected content
```

Comment:
I don't think this is scalable tho, in the past we've used crawlers to check for URL checking

Reply:
It's a big job for an entire site or section, but it shouldn't be over one ticket. I check each link as I add it to the content I'm creating so this is easy for me to check off. At some point in the content creation process, the writer has to make sure the link works.